include/net/ipv4_autoconf.h

```diff
@@ -0,0 +1,65 @@
+/** @file
+ * @brief IPv4 Autoconfiguration
+ */
+
+/*
+ * Copyright (c) 2017 Matthias Boesl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
You could simplify the license boiler plate text here by just saying "SPDX-License-Identifier: Apache-2.0", like the other header files are doing.